flink/src/main/java/org/apache/iceberg/flink/connector/IcebergConnectorConstant.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.flink.connector;
+
+import java.util.concurrent.TimeUnit;
+import org.apache.hadoop.hive.conf.HiveConf.ConfVars;
+
+public class IcebergConnectorConstant {
+  private IcebergConnectorConstant() {}
+
+  public static final String TYPE = "iceberg";
+
+  public static final String ICEBERG_APP_TYPE = "flink";
+
+  //public static final String CATALOG = "catalog";
+  public static final String DATABASE = "database";
+  public static final String TABLE = "table";
+
+  public static final String FORMAT = "format";
+  public static final String CODEC = "codec";
+
+  public static final String METACAT_HOST_HADOOP_CONF_KEY = ConfVars.METASTOREURIS.varname;  // For HiveCatalog
+  public static final String METACAT_HOST = "metacat_host";
```

Comment:
Metacat is the Netflix metastore. This should probably be rewritten to use HiveCatalog.